woocommerce/woocommerce
# Make state optional for BG and HU

### All Submissions:






### Changes proposed in this Pull Request:



This creates issues with Apple/Google Pay, so removing the requirement to include state in the HU and BG address.






### How to test the changes in this Pull Request:



*Please also see the comment below *

1. Create a WC store located in any country, e.g. in the UK, selling to at least HU and BG (but can sell to all countries of course)
2. Activate a simple payment method such as payment on delivery.
3. Add a simple product called "Uncountable widget" with price 10 GBP.
4. Add the product to cart, then go to checkout and set the billing and shipping country to `Hungary`. 
5. You should be able to check out without providing the State info.
6. Go back to Shop, add the "Uncountable widget" product to the cart again
7. Go to checkout
8. Choose `Bulgaria` as Billing Country
9. You should be able to check out without providing State info.



### Other information:




### FOR PR REVIEWER ONLY:
